python/ray/tests/test_advanced_2.py

```diff
@@ -183,18 +183,25 @@ def method(self):
 
     del f1, f3
 
-    # Make sure that we get exceptions if we submit tasks that require a
-    # fractional number of resources greater than 1.
+    # Non unit resources (e.g. CPU, ) allow fractional
+    # number of resources greather than 1.
+    @ray.remote(num_cpus=1.5, resources={"Custom": 2.5})
+    def test_frac_cpu():
+        pass
 
-    @ray.remote(num_cpus=1.5)
-    def test():
+    test_frac_cpu.remote()
```

Comment:
```suggestion
    assert ray.get(test_frac_cpu.remote())
```

and make test_frac_cpu return True (it is better having explicit assert)